src/components/filter/category/CategoryItem.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import { defineComponent, ref, watch } from 'vue'
+import { useRouter, useRoute } from 'vue-router'
+
+export default defineComponent({
+    name: 'CategoryItem',
+    props: {
+        label: { type: String, required: true },
+        count: { type: String, required: true },
+        childrenItems: {
+            type: Array as () => Array<{ label: string; childrenItems?: any[] }>,
+            default: () => []
+        }
```

Comment:
```suggestion
        childrenItems: {
            type: Array as () => Array<{ label: string; childrenItems?: any[] }>,
            default: () => []
        },
        parentActiveMenu: {
            type: Function,
            default() {
                return () => {}
            }
        }
```
Bổ sung thêm 1 props là func để active parent category